constants.py

```diff
@@ -791,3 +791,34 @@
 
 # radians
 kShootingAngleFudgeAmount = 0.01
+# Elevator constants, replace values
+
+kElevator1CANID = 60
+kElevator1Name = "Elevator1Motor"
+kElevator1PGain = 0.12
+kElevator1IGain = 0
+kElevator1DGain = 0
+kElevator1Inverted = False
+
+kElevator2CANID = 60
+kElevator2Name = "Elevator2Motor"
+kElevator2PGain = 0.12
+kElevator2IGain = 0
+kElevator2DGain = 0
+kElevator2Inverted = True
+
+kMotorPulleyGearRatio = 60 / 18
```

Comment:
is this the correct ratio? iirc theres also a versaplanetary in there
total ratio would be (60/18) * (4/1)